src/main/java/pro/sky/coursework2/service/ExaminerServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package pro.sky.coursework2.service;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import pro.sky.coursework2.exceptions.InvalidInputException;
+import pro.sky.coursework2.model.Question;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Service
+public class ExaminerServiceImpl implements ExaminerService {
+
+    private final QuestionService javaService;
+    private final QuestionService mathService;
+
+    public ExaminerServiceImpl(@Qualifier("javaQuestionService") QuestionService javaService,
+                               @Qualifier("mathQuestionService") QuestionService mathService) {
```

Comment:
```suggestion
    public ExaminerServiceImpl(List<QuestionService> serviceList) {
```
Вот так можно заинжектить все реализации QuestionService